tests/deployment/multi_networks.py

```diff
@@ -0,0 +1,152 @@
+"""Test various properties re: vlan tagged networks, on real hardware.
+
+For guidance on running these tests, see the section on deployment
+tests in docs/testing.md
+"""
+
+import json
+
+from hil import api, model, deferred, config
+from hil.test_common import config_testsuite, fail_on_log_warnings, \
+    fresh_database, with_request_context, site_layout, NetworkTest, \
+    network_create_simple, server_init
+import pytest
+
+
+@pytest.fixture
+def configure():
+    """Configure HIL."""
+    config_testsuite()
+    config.load_extensions()
+
+
+fail_on_log_warnings = pytest.fixture(autouse=True)(fail_on_log_warnings)
+fresh_database = pytest.fixture(fresh_database)
+server_init = pytest.fixture(server_init)
+
+
+with_request_context = pytest.yield_fixture(with_request_context)
+
+site_layout = pytest.fixture(site_layout)
+
+pytestmark = pytest.mark.usefixtures('configure',
+                                     'server_init',
+                                     'fresh_database',
+                                     'with_request_context',
+                                     'site_layout')
+
+
+class TestMultiNets(NetworkTest):
+    """NetworkTest using multiple tagged vlan networks on a port."""
+
+    def test_multi_networks(self):
+        """Do a bunch of network operations on the switch, and verify things
+        along the way.
+
+        The above is super vague; unfortunately the setup operations are very
+        slow, so it makes a huge difference to do everything in one pass. See
+        the comments in-line to understand exactly what is being tested.
```

Comment:
We've now got this exact docstring copied in 3 places. I'm worried about how much copypasta I've been seing in patches to the test suite lately...